package.json

```
@@ -207,22 +151,106 @@
         "!packages/{components,themes}/*/node_modules/**"
       ]
     },
-    "lint:components": {
+    "dist": {
+      "dependencies": [
+        "build",
+        "dist:checklist",
+        "dist:components",
+        "dist:themes"
+      ]
+    },
+    "dist:checklist": {
+      "command": "esbuild --minify --bundle packages/checklist/index.ts --outdir=packages/checklist/dist --sourcemap",
+      "dependencies": [
+        "dist:checklist:code"
+      ],
+      "files": [
+        "packages/checklist/**/*.scss",
+        "packages/checklist/**/*.ts",
+        "!packages/checklist/**/*.d.ts"
+      ],
+      "output": [
+        "packages/checklist/dist/index.js",
+        "packages/checklist/dist/index.js.map"
+      ]
+    },
+    "dist:checklist:code": {
+      "command": "node scripts/build-packages.js checklist",
+      "clean": "if-file-deleted",
+      "dependencies": [
+        "dist:checklist:styles"
+      ],
+      "files": [
+        "packages/checklist/**/*.ts",
+        "!packages/checklist/**/*.d.ts"
+      ],
+      "output": [
+        "packages/checklist/**/*.js",
+        "packages/checklist/**/*.js.map"
+      ]
+    },
+    "dist:checklist:styles": {
+      "command": "node scripts/build-styles.js \"packages/checklist/**/*.scss\"",
+      "clean": "if-file-deleted",
+      "dependencies": [
+        "dist:themes"
+      ],
+      "output": [
+        "packages/checklist/**/*.scss.ts"
+      ]
+    },
+    "dist:components": {
+      "command": "node scripts/build-metadata.js",
+      "clean": "if-file-deleted",
+      "dependencies": [
+        "build:components:styles"
+      ],
+      "files": [
+        "packages/components/**/*.ts",
+        "!packages/components/**/*.{d,spec,stories}.ts"
+      ],
+      "output": [
+        "packages/components/*/custom-elements.json"
+      ]
+    },
+    "dist:themes": {
+      "command": "node scripts/build-themes.js --production",
+      "clean": "if-file-deleted",
+      "output": [
+        "packages/themes/**/*.{css,scss}",
+        "!packages/themes/*/fonts.css"
+      ]
+    },
+    "import-icons": {
+      "command": "node scripts/import-icons.js",
+      "clean": false,
+      "files": [
+        "scripts/figma-icon-pages.json",
+        "scripts/import-icons.js",
+        "packages/tokens/src/core.json",
+        "packages/tokens/src/*/base.json"
+      ],
+      "output": [
+        "packages/themes/*/icons.ts",
+        "packages/themes/*/icons/*.svg"
+      ]
+    },
+    "lint": {
       "dependencies": [
         "lint:components:code",
         "lint:components:styles"
       ]
     },
     "lint:components:code": {
-      "command": "eslint --config packages/components/eslint.config.mjs packages/components/*"
+      "command": "eslint --config packages/components/eslint.config.mjs \"packages/components/**/*.ts\""
```

So maybe better without escaping?

```suggestion
      "command": "eslint --config packages/components/eslint.config.mjs 'packages/components/**/*.ts'"
```